Feat : Updated Import/Export Page

Mahesh-Umachagi opened the pull request:
-   [x] Have you followed the [contributing guidelines](https://github.com/ever-co/gauzy/blob/master/.github/CONTRIBUTING.md)?
-   [x] Have you explained what your changes do, and why they add value?

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

---

evereq commented:
@Mahesh-Umachagi , I don't see "Added all the entities to Templates" in the code changes at all, did you pushed that part already?

Mahesh-Umachagi commented:
> @Mahesh-Umachagi , I don't see "Added all the entities to Templates" in the code changes at all, did you pushed that part already?

Hi @evereq ! Sorry you won't see the template files in the code change because the templates are stored as CSV files which are all zipped in the file export/template.zip, hence you will only see that one single zip file 🙂

![Screenshot (17)](https://user-images.githubusercontent.com/58215972/101395577-26408080-38f0-11eb-997f-cb3ea5e0d6a5.png)

![Screenshot (20)](https://user-images.githubusercontent.com/58215972/101395731-5851e280-38f0-11eb-89e3-e9970c34205a.png)

evereq commented:
@Mahesh-Umachagi oh yes, I misunderstand it initially.. I thought about all entities to be used in the exporting/importing service! I.e. that if I click "Export" it might export ALL entities in DB because many of the entities added recently probably not included right now. The same with Import, etc. Does it make sense? Can you do it next? Also, think we should add such entities selection in UI (where we have lots of checkboxes). We currently have 160+ tables in DB, I doubt we display all of them now? (we may need to somehow improve the design as 160 is a lot and we expect even more :D)

evereq commented:
@Mahesh-Umachagi  P.S. I don't see you pushed any changed to that Zip file, did you?

Mahesh-Umachagi commented:
@evereq  Sure yes, I will get to that soon.. 

I changed quite a lot of things in the CSV templates.
1. Earlier there were ~70 files and now I've added 91 more so we have 161 files 🙂 
2. I also went through all the tables vs. files and added the entities which were not in the files but were there in the tables.

Earlier Template :
![Screenshot (23)](https://user-images.githubusercontent.com/58215972/101444990-ae517500-3946-11eb-8d9d-f1c60dca1bb8.png)

![Screenshot (25)](https://user-images.githubusercontent.com/58215972/101444995-b01b3880-3946-11eb-9fd7-20af9c79e69f.png)

![Screenshot (28)](https://user-images.githubusercontent.com/58215972/101444997-b0b3cf00-3946-11eb-86e9-ef066ce6c44c.png)

Updated Template :

Added Files
![Screenshot (20)](https://user-images.githubusercontent.com/58215972/101445189-0b4d2b00-3947-11eb-876e-7fc95af3b47e.png)

Updated Columns
![Screenshot (26)](https://user-images.githubusercontent.com/58215972/101445193-0c7e5800-3947-11eb-8c68-08976534a37e.png)

Updated  File names
![Screenshot (29)](https://user-images.githubusercontent.com/58215972/101445195-0d16ee80-3947-11eb-84c0-ab4a28d4074c.png)

evereq commented:
@Mahesh-Umachagi sure I understand it's a lot of work. I wonder, how did you make that template files? I thought we just do an export of all data (into CSV files) and next clean up such CSV files to preserve the only structure, but no records and finally just zip it all. What is your flow to make it? I hope you are not doing it all by hand...

Mahesh-Umachagi commented:
@evereq I have Exported the files from PGadmin to CSV format, Sorry I just meant a lot of file changes, but yes that's how I did it :blush:
